src/components/Layout/HeaderDropDown.js

```diff
@@ -30,14 +30,19 @@ class HeaderDropDown extends React.Component {
 
   render () {
     return (
-      <div className={cx(this.props.className, style.dropDownItem)}>
+      <a className={cx(this.props.className, style.dropDownItem)}>
```

Comment:
replaced it with span - should work better since span is - like a - an "inline tag" and thus closer to the defaults of a